kt/godot-runtime/src/main/kotlin/godot/core/pool/PoolByteArray.kt

```diff
@@ -8,12 +8,35 @@ import godot.util.VoidPtr
 @Suppress("MemberVisibilityCanBePrivate", "unused")
 class PoolByteArray : NativeCoreType, Iterable<Byte> {
 
+    enum class CompressionMode {
+        /**
+         * Uses the FastLZ compression method.
+         */
+        COMPRESSION_FASTLZ,
+
+        /**
+         * Uses the DEFLATE compression method.
+         */
+        COMPRESSION_DEFLATE,
+
+        /**
+         * Uses the Zstandard compression method.
+         */
+        COMPRESSION_ZSTD,
+
+        /**
+         * Uses the gzip compression method.
+         */
+        COMPRESSION_GZIP
+    }
+
+

```

Comment:
Hmm isn't the idea that the compression modes in `File` class are used? At least thats to where the link [CompressionMode](https://docs.godotengine.org/en/stable/classes/class_file.html#enum-file-compressionmode) in https://docs.godotengine.org/en/stable/classes/class_poolbytearray.html#class-poolbytearray-method-compress is linking.

Reply:
Yeah, it is in File. 
But PoolByteArray is a different module than the GenAPI. I can't access it from there.

Reply:
So maybe a comment here would help ?